index.html

```diff
@@ -0,0 +1,1316 @@
+                </a>
+                </h5>
+            </div>
+          </div> 
+        </div>
+        <div class="col mb-4">
+          <div class="card border-0">
+            <img src="./images/sport2.webp" class="card-img-top card-header-img mb-2" alt="sports image 2">
+            <div class="card-body m-0 p-0">
+              <h5 class="card-title mt-2">
+                <a href="#" class="badge-light">
+                  Joe Kelly's Mocking of Houston Astros Before All-In Brawl Goes Viral
+                </a>
+              </h5>
+            </div>
+          </div> 
+        </div>
+        <div class="col mb-4">
+          <div class="media">
+            <img class="mr-3 card-header-img" src="./images/usmedia3.webp" alt="Generic placeholder image">
+            <div class="media-body">
+              <h5 class="mt-0">
+                <a href="#" class="badge-light">
+                  Here Are Kobe Bryant's '10 Rules' Painted on Philadelphia Eagles Facility
+                </a>
+              </h5>
+            </div>
+          </div>
+          <hr>
+          <div class="media">
+            <img class="mr-3" src="./images/sport4.webp" alt="Generic placeholder image">
+            <div class="media-body">
+              <h5 class="mt-0">
+                <a href="#" class="badge-light">
+                  'Last Chance U' Season 5: Start Date, Schedule, Where to Watch, Stream
+                </a>
+              </h5>        
+            </div>
+          </div>
+          <hr>
+          <div class="media">
+            <img class="mr-3 " src="./images/sport5.webp" alt="Generic placeholder image">
+            <div class="media-body">
+              <h5 class="mt-0">
+                <a href="#" class="badge-light">
+                  What Mike Tyson's Opponent Has Said About Fighting Boxing Legend
+                </a>
+              </h5>
+            </div>
+          </div>
+        </div>        
+      </div>
+    </section>
+    <hr>
+
+    <section id="Promo" class="container-lg mt-5">
+      <div class="row row-cols-1 row-cols-lg-2">
+        <div class="col-lg-7 col-xl-8 mb-4">
+          <p class="text-news mb-3 pl-0 font-weight-bolder">SUBSCRIBE</p>
+          <div class="card mb-3 bg-light border-0">
+            <div class="row no-gutters">
+              <div class="col-md-4">
+                <img src="./images/subscribe-image.jpg" class="card-img" alt="subscribe">
+              </div>
+              <div class="col-md-8">
+                <div class="card-body pt-5">
+                  <h5 class="card-title font-weight-bolder">
+                    In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h5>
+                  <p class="card-text"><small class="text-muted">All Access + Weekly Delivery.<br>Choose A Membership That's Perfect for You!</small></p>
+                  <button type="button" class="btn btn-danger">SUBSCRIBE ></button>
```

Comment:
#### Small screen size

- [ ] The image in the subscribe section is too big. The newsletter section should not have an image.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88951714-c0225d80-d26c-11ea-86d5-c7a58ac0bab8.png)|![image](https://user-images.githubusercontent.com/61479648/88951780-d3352d80-d26c-11ea-99e8-dc20f8d71817.png)|

#### General

- [ ] The resolution of this image seems to be pretty bad because the height is not proportional to the width. Was this in the original? If yes, submit a print screen for the next review or change to the original it is not the case.

![image](https://user-images.githubusercontent.com/61479648/88952006-2a3b0280-d26d-11ea-9e31-bc98df730709.png)